### ISSUE TITLE
Trackpad Acceleration 

### ISSUE THREAD

**Issue by chino0217** [linearmouse/linearmouse#465]
### OS

macOS 13.1 (Build 22C65)

### LinearMouse

v0.9.0

### Is your feature request related to a problem?

I'm in love with everything this software provides, no bugs or issue since installing.issues

### Describe the solution you'd like

For Mac users alike that have always used mouse acceleration through the trackpad and mouse (even when disabling through the terminal is not enough) a feature to support acceleration on the trackpad while being able to disable for mouse. 

Basically a feature to keep acceleration on trackpad while disabling on mouse.

**Comment by lujjjh** [comment]
Thanks for your feedback!

LinearMouse allows you to customize settings for each device independently. Therefore, if you disable acceleration for your mouse, it should not affect the trackpad.

However, if you changed the settings with your trackpad, it's possible that you accidentally disabled acceleration for the trackpad as well. 

Similar issue: https://github.com/linearmouse/linearmouse/issues/210#issuecomment-1199010477
